echopype/filter/api.py

```diff
@@ -0,0 +1,72 @@
+"""
+Functions for enhancing the spatial and temporal coherence of data.
```

Comment:
I think this is the description to use for the `unify` subpackage. 
I would suggest replacing this with "Functions for reducing variabilities in backscatter data."

Reply:
Thanks for catching that and offering a replacement. It was an oversight. I'll push a commit with the change.

Reply:
Done.